diesel_compile_tests/tests/compile-fail/select_for_update_skip_locked_cannot_be_used_on_mysql.rs

```diff
@@ -0,0 +1,21 @@
+#[macro_use]
+extern crate diesel;
+
+use diesel::*;
+use diesel::mysql::MysqlConnection;
+
+table! {
+    users {
+        id -> Integer,
+    }
+}
+
+fn main() {
+    let conn = MysqlConnection::establish("").unwrap();
+    users::table
+        .for_update()
+        .skip_locked()
+        .load(&conn)
+        //~^ ERROR: E0271
```

Comment:
MySQL 8 does support these modifiers, so we should allow them on that backend.

Reply:
How do you handle that right now? Should I just enable it outright for the MySQL back-end, or is there a way to specify the minimum supported MySQL version at compile time?

Reply:
I'd just enable it outright for the backend, and put a note in the docs about the supported versions.

Reply:
We should mention that `SKIP LOCKED` is only on PG 9.5+ as well